packages/roosterjs-editor-api/lib/format/cacheGetListElement.ts

```diff
@@ -5,19 +5,26 @@ import { getTagOfNode } from 'roosterjs-editor-dom';
 
 const EVENTDATACACHE_LISTELEMENT = 'LISTELEMENT';
 
-export default function cacheGetListElement(editor: Editor, event?: PluginEvent): Element {
+export default function cacheGetListElement(editor: Editor, event?: PluginEvent, listType?: string): Element {
     return cacheGetEventData<Element>(event, EVENTDATACACHE_LISTELEMENT, () => {
         let node = getNodeAtCursor(editor);
-        let startElement =
-            node && node.nodeType == NodeType.Text ? node.parentElement : <Element>node;
-        while (startElement && editor.contains(startElement)) {
-            let tagName = getTagOfNode(startElement);
-            if (tagName == 'LI') {
-                return startElement;
-            }
-            startElement = startElement.parentElement;
+        if (!listType) {
+            listType = 'LI';
```

Comment:
Suggest to change to a simple way: listType = listType || 'LI";

Reply:
Changed.